examples/interactive/cli-platform/src/lib.rs

```diff
@@ -78,11 +78,14 @@ pub extern "C" fn rust_main() -> i32 {
     let size = unsafe { roc_main_size() } as usize;
     let layout = Layout::array::<u8>(size).unwrap();
 
+    // TODO: can we be more efficient about reusing the String's memory for RocStr?
+    let args: RocList<RocStr> = std::env::args().map(|s| RocStr::from(s.as_str())).collect();
```

Comment:
Might be worth using `env::args_os` and `to_string_lossy` to gracefully translate invalid Unicode chars into the unicode replacement character. (`env::args()` panics if it encounters any of those.)

```rust
        let args: RocList<RocStr> = std::env::args_os()
            .map(|os_string| RocStr::from(os_string.to_string_lossy().borrow()))
            .collect();
```

Reply:
Good note, didn’t know args() panics!